requirements.txt

```diff
@@ -1,4 +1,4 @@
 fyle>=v0.28.0
-fyle_accounting_mappings==1.7.0
+fyle_accounting_mappings==1.25.0
```

Comment:
we'll need to update the version in setup.py as well

Reply:
this is already done no check the above file

Reply:
arey bhai, fyle accounting mappings version change karo, in `install_requires`